src/Std/Time/Time/PlainTime.lean

```diff
@@ -30,213 +30,225 @@ structure PlainTime where
   /--
   `Second` component of the `PlainTime`
   -/
-  second : Sigma Second.Ordinal
+  second : Second.Ordinal leap
 
   /--
   `Nanoseconds` component of the `PlainTime`
   -/
   nanosecond : Nanosecond.Ordinal
   deriving Repr
 
-instance : Inhabited PlainTime where
-  default := ⟨0, 0, Sigma.mk false 0, 0, by decide⟩
+/--
+COMMENT
+-/
+def LeapTime := PlainTime True
```

Comment:
```suggestion
abbrev LeapTime := PlainTime True
```

I imagine this is still TODO (given the `COMMENT`), but I imagine this definitely should be an `abbrev` rather than a `def` to inherit the instances of `PlainTime`.